### ISSUE TITLE
Traceback when running from command line on linux

### ISSUE THREAD

**Issue by yejingchen** [CacheBrowser/cachebrowser#16]
I cloned the code from GitHub, and used `sudo python setup.py install` to install. But when I set browser's proxy to `localhost:8080` and ran `cachebrowser` it didn't seem to work properly. A traceback was given everytime I wanted to open a new web page.

I am on Fedora 23 x86_64, Firefox 42.0, Python 2.7.10.
`uname -a` gives `Linux localhost 4.2.6-301.fc23.x86_64 #1 SMP Fri Nov 20 22:22:41 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`

```
$ cachebrowser
('SELECT name FROM sqlite_master WHERE type = ? ORDER BY name;', ('table',))
('SELECT name FROM sqlite_master WHERE type = ? ORDER BY name;', ('table',))
Cachebrowser running...
Waiting for connections...
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/cachebrowser/common.py", line 19, in inner
    return func(*args, **kwargs)
  File "build/bdist.linux-x86_64/egg/cachebrowser/proxy.py", line 45, in on_local_data
    self._schema = schema(self, self._buffer)
  File "build/bdist.linux-x86_64/egg/cachebrowser/proxy.py", line 173, in __init__
    self._start_upstream()
  File "build/bdist.linux-x86_64/egg/cachebrowser/proxy.py", line 201, in _start_upstream
    Host.get(url=host)
  File "/usr/lib/python2.7/site-packages/peewee-2.7.3-py2.7.egg/peewee.py", line 4392, in get
    sq = sq.filter(**kwargs)
  File "/usr/lib/python2.7/site-packages/peewee-2.7.3-py2.7.egg/peewee.py", line 2524, in filter
    query, joins = self.convert_dict_to_node(piece.query)
  File "/usr/lib/python2.7/site-packages/peewee-2.7.3-py2.7.egg/peewee.py", line 2500, in convert_dict_to_node
    model_attr = getattr(curr, piece)
AttributeError: type object 'Host' has no attribute 'url'

(The same trackback appeared again and again so I just skipped them here)
```

**Comment by elerain** [comment]
Same question!

**Comment by ha-D** [comment]
Fixed 637868100914f781dcf4b616fd9af5814d8ebbe4
